libr/core/cmd_type.c

```diff
@@ -795,7 +885,8 @@ static int cmd_type(void *data, const char *input) {
 		}
 		break;
 	case 'p': { // "tp"
-		char *ptr = r_str_trim (strdup (input + 1));
+		char *tmp = strdup (input);
+		char *ptr = r_str_trim (strchr (tmp , ' '));
```

Comment:
i was thinking yesterday in adding an r_str_trimdup() but it will be good to have a proper discussion of all the r_util apis

Reply:
yup r_str_trimdup() could save us a few lines :)